src/diffusers/pipelines/controlnet/pipeline_controlnet_img2img.py

```diff
@@ -915,14 +899,12 @@ def __call__(
             (nsfw) content, according to the `safety_checker`.
         """
         # 0. Default height and width to unet
-        height, width = self._default_height_width(height, width, image)
+        # height, width = self._default_height_width(height, width, image)
```

Comment:
I think we don't need this `default_height_width` function anymore because the `image_processor.preprocess` will resize the image to default size